### DISCUSSION TITLE
Problem with switch theme

### DISCUSSION THREAD

**Discussion by mrcode0x0** [Nicasiomarques/github-user-search-app#13] [Ideas]
Quando ao switch theme reparei que tem um bug quando o systema operativo esta no light mode, o botão troca o restante dos elementos mas não troca a esquema de cores.

sugiro esse post talvez possa ajudar

**Comment by Nicasiomarques** [comment]
Thank you very much, we will take it into account, could you please open an issue explaining more details.
